### ISSUE TITLE
Breakpoint inline edit button should be a submenu

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#115111]
Testing #114345

I can set an expression and a hit count on function breakpoints:

![image](https://user-images.githubusercontent.com/2644648/105884733-1fafba80-5fbd-11eb-9764-31060c48ad12.png)

but the ✏️ button only changes the _expression_.

If there is only 1 button, I would have expected it to either:
1. edit the function name
2. only hsve 1 button if _either_ expression _or_ hit count was set and 2 buttons if _both_ expression _and_ hit count were set (the second button to edit the hit count)

As it stands I have to realize that the edit button only edits the expression and then cancel and right click if I want to edit the hit count or function name.

**Comment by isidorn** [comment]
@TylerLeonhardt I like the dea of having two buttons, however would you expect different icons? Since I think showing two of the same icon does not really work, or does it?

fyi @misolori

**Comment by TylerLeonhardt** [comment]
> however would you expect different icons?

Yes I'd expect different icons. I don't have any suggestions for what they should look like though... maybe an expression is something like `[?]✎` but then `[#]✎` is hit count? (where the pencil is in front of a piece of paper with `?` or `#` on it)

**Comment by TylerLeonhardt** [comment]
might need a 3rd button for log points... not sure.

**Comment by gjsjohnmurray** [comment]
What about giving the edit button an alt command as well? With #101995 implemented the tooltip would convey both.

IIRC there was a recent request for these buttons to support a dropdown list of commands from which to pick. Pinging @eamodio as I think it may have come from him.

**Comment by misolori** [comment]
Could the edit button show a dropdown with the edit options? I'd really like to avoid making icons complicated as they will slow the user down as they parse what it means.

**Comment by eamodio** [comment]
With the recent submenu work, we can support these actions being drop down menus.

**Comment by isidorn** [comment]
@misolori Yeah I like that idea!
Assigning optimisticaly to January and might push out to february

**Comment by isidorn** [comment]
Pushing this out to next milestone since I did not have time for this in endgame.
fyi @weinand and @connor4312 for the submenu idea

**Comment by weinand** [comment]
@isidorn yes, let's try a submenu.

**Comment by isidorn** [comment]
I went with the submenu approach.

However for regular breakpoint I decided to go with single click -> open in editor. Since there the dropdown is present to change what you are editing. If you think that having a submenu for regular breakpoints is also nice let me know and I can change it there as well.

![Screenshot 2021-03-09 at 15 23 04](https://user-images.githubusercontent.com/1926584/110484873-5cf28680-80eb-11eb-98b3-06eae2f026be.png)
